data_handling/munge_ansirh.py

```diff
@@ -0,0 +1,109 @@
+"""
+Munge ANSIRH data into JSON files binned by state.
+"""
+
+import pandas as pd
+import json
+from .clean_ansirh import clean_ansirh
+
+
+def main_ansirh():
+    """
+    Create state dictionary of data from ANSIRH.
+    """
+
+    ansirh = pd.read_csv("./data/AFD_2021_for_ArcGIS_Upload.csv")
+
+    # drop empty column
+    ansirh = ansirh.drop(["Unnamed: 2"], axis=1)
+
+    # make dictionaries of every row in dataset
+    row_dicts = make_row_dicts(ansirh)
+
+    # clean data
+    clean_row_dicts = clean_ansirh(row_dicts)
+
+    # sort by state
+    state_dict = split_by_state(clean_row_dicts)
+
+    # sort by zipcode
+    zip_dict = split_by_zip(state_dict)
+
+    # TODO: write to JSON
+    with open("data/clean_ansirh.json", "w", encoding="utf-8") as outfile:
+        json.dump(zip_dict, outfile, indent=1)
+
+    return zip_dict
+
+
+def split_by_state(rows):
+    """
+    Create dictionary of states containing list of row dictionaries
+    """
+
+    state_dict = {}
+    for row in rows:
+
+        # translate two letter state code to full name
+        full_state = str(translate_code_to_state(row["state"]))
+
+        # create states dictionary with row dicts list as value
+        if full_state not in state_dict:
+            # print(f"{full_state}: {type(full_state)}")
+            state_dict[full_state] = [row]
+        else:
+            state_dict[full_state].append(row)
+
+    return state_dict
+
+
+def translate_code_to_state(state_abr):
+    """
+    Turns two-letter state code into the full state name.
+    """
+
+    # read in state abreviation data
+    file_name = "./data/state_abbreviations.csv"
+    state = pd.read_csv(file_name)
```

Comment:
Why `pandas` instead of `csv.DictReader`?

Reply:
for the vectorization of it all. And also why not?

Reply:
`pandas` is primarily for data analysis and we are just using it for handing a `csv` and cleaning, which I don't think necessarily requires `pandas`. It feels like we're over-engineering this process if we're using a whole non-standard library for something a standard library could easily do (you in theory don't want to include more external packages than are necessary as it complicates building the program, etc.). 

If you are going to use `pandas` though, make sure to add it via `poetry add pandas` as I don't see it in the `pyproject.toml`. I assume that it's taking the imported pandas from your base python version right now.

Reply:
If you'd rather not do that NOW, we'd just need to do the `poetry add pandas` thing and put a comment above it saying `# TODO: Will refactor with csv in the near future`.